src/modules/infobox/components/FeatureDetail.vue

```diff
@@ -49,6 +51,18 @@ function sanitizeHtml(htmlText) {
                 {{ i18n.t('no_more_information') }}
             </div>
         </div>
+        <div
+            v-if="feature.geometry.type == 'Point'"
```

Comment:
shouldn't be too much of an issue here, but you typically should use a triple equals in JS, so that it also type-checks (and not only try to guess-work the values and compare them in all way possible)